.gitignore

```diff
@@ -4,6 +4,6 @@ _build
 lib/*.js
 samples/*.js
 samples/*.txt
-test/*.js
```

Comment:
Looks like gitignore didn't get updated after rearranging the tests folder. Updated for convenience

Reply:
Can we just get rid of testoutput.txt?